projects/S905/packages/brcmfmac_sdio-firmware-aml/package.mk

```diff
@@ -0,0 +1,40 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="brcmfmac_sdio-firmware-aml"
```

Comment:
why not use/change https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/linux-firmware/brcmfmac_sdio-firmware-aml/package.mk ?

Reply:
Have you seen how many more firmwares there are in kszaqs repo?